### PR TITLE
etcdctl: print messages for successful auth operations

### PR DESCRIPTION
**Pull request by mitake**
This commit lets etcdctl v3 follow the manner of etcdctl v2.

### CODE REVIEW

**Comment by xiang90** [comment]
lgtm

File: etcdctl/ctlv3/command/auth_command.go [hunk 1]
```diff
@@ -52,4 +52,6 @@ func authEnableCommandFunc(cmd *cobra.Command, args []string) {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
+
+	fmt.Printf("Authentication Enabled")
```
Comment (gyuho) [inline etcdctl/ctlv3/command/auth_command.go]: `fmt.Println`?

Comment (mitake) [inline etcdctl/ctlv3/command/auth_command.go]: Ah, thanks for catching!

**Comment by mitake** [comment]
@gyuho @xiang90 updated for the `fmt.Println()`.

File: etcdctl/ctlv3/command/auth_command.go (hunk 1 above)
Comment (gyuho) [inline etcdctl/ctlv3/command/auth_command.go]: lgtm. thanks!

**Pull request merged**